bash_completion

```diff
@@ -1620,7 +1620,7 @@ _comp_compgen_mac_addresses()
     # - ip link: link/ether
     _comp_compgen -v addresses split -- "$(
         {
-            LC_ALL=C ifconfig -a || ip -c=never link show || ip link show
+            LC_ALL=C ip -c=never link show || ip link show || ifconfig -a
```

Comment:
This makes LC_ALL=C apply to the `ip` command and removes it from the `ifconfig`. I assume that is not on purpose; if it is, I think it deserves a comment.